### ISSUE TITLE
Python 2.7 Depreciation

### ISSUE THREAD

**Issue by JeffResc** [Screenly/screenly-ose#1045]
When installing, this error is given:
```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
```
I have not tested screenly-ose with other versions of Python, but I believe that screenly-ose needs to be updated to future versions of Python before January 1st, 2020.

**Comment by vpetersson** [comment]
This is a known issue. Upgrading to Python 3 is a fairly sizeable task (partially due to dependencies that may yet not be compatible).

**Comment by vpetersson** [comment]
Closed in favour of #1091.
